apps/node-red/data/settings.js

```diff
@@ -0,0 +1,497 @@
+/**
```

Comment:
Do we need to include this file?

If I remove it it gets auto created on start, so probably best we just let whichever version of Node-RED we're shipping dump it's default config than worry about keeping them both in sync when we update.

Reply:
(answered in DMs, yes we do)